code/modules/mapping/map_template.dm

```diff
@@ -8,6 +8,12 @@
 	var/keep_cached_map = FALSE
 	var/station_id = null // used to override the root id when generating
 
+	var/should_place_on_top = TRUE
+
+	var/returns_created_atoms = FALSE
+	var/list/created_atoms = list()
+	var/list/turf_blacklist = list()
```

Comment:
This blacklist doesn't seem to need to live outside of the load call.

Reply:
its now created entirely in map_template/load() and passed to parsed